doc/development/setup.rst

```diff
@@ -54,6 +55,12 @@ Working with the code
 The first thing you need are all the main application's dependencies::
 
     cd src/
+    url -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+    sudo apt install nodejs
+    url -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
```

Comment:
What is nvm and does one actually need that?

Reply:
nvm = node version manager, can be omitted by simply installing the current node version above